README.md

```diff
@@ -128,7 +128,7 @@ Here's the whole workflow we're targeting:
 ```yaml
 ---
 name: CI
-'on':
+on:
```

Comment:
Can you leave the `'on'` for now? `yamllint` actually identifies that as an ambiguous identifier (indeed, some YAML parsers have trouble with it and identify it as a bool instead of a string).